app.json

```diff
@@ -33,9 +33,9 @@
      */
     "classpath": [
         "app",
-        "./lib/geoext3/src",
```

Comment:
Maybe just comment out these ./lib references. It would be useful to be able to still clone the repos to these locations to debug against the main branches of dependencies. A few times during cpsi-mapview development there were minor fixes that had to be added to GeoExt, and being able to debug and then push a pull request from here would help development.